packages/code/src/types.ts

```diff
@@ -50,6 +50,11 @@ export interface SyntaxProps extends HTMLElementProps<'code'> {
    * An array of lines to highlight. The array can only contain numbers corresponding to the line numbers to highlight, and / or tuples representing a range (e.g. `[6, 10]`);
    */
   highlightLines?: LineHighlightingDefinition;
+
+  /**
+   * A ref to the underlying `code` element.
+   */
+  ref?: React.Ref<HTMLPreElement>;
```

Comment:
You can remove this line since you're using `ComponentPropsWithRef`

Reply:
On second thought, this is no longer needed at all. I was originally adding a `ref` to access a `Sytnax` child, but found a way around this by using another reference. I'm going to revert these changes entirely.